server/conf/application.conf

```diff
@@ -655,7 +655,8 @@ allow_civiform_admin_access_programs = ${?ALLOW_CIVIFORM_ADMIN_ACCESS_PROGRAMS}
 # Preview features.
 
 # In development features.
-application_status_tracking_enabled = false
+# Status Tracking is a default feature as of Jan 2023, this control is deprecated.
+application_status_tracking_enabled = true
 application_status_tracking_enabled = ${?CIVIFORM_APPLICATION_STATUS_TRACKING_ENABLED}
```

Comment:
question (non-blocking): why not remove this config value altogether?

Reply:
I seem to recall that removing this or maybe CIVIFORM_APPLICATION_STATUS_TRACKING_ENABLED , if deployment scripts still use them can cause issues.  Basically that all references need to be removed first.

Does that sound right?

Reply:
Not that I can remember, doesn't hurt to take this approach though